common/i18n/resources/en.js

```diff
@@ -246,10 +247,10 @@ module.exports = {
     TaxonomyImportJob: 'Taxonomy Import',
   },
 
-  formErrors: {
+  errors: {
```

Comment:
let's keep separate formErrors to systemErrors (as agreed)

Reply:
Ok